.nvmrc

```diff
@@ -0,0 +1 @@
+v16
```

Comment:
Provides a lock for nvm to use so all developers are not using different version of npm.